frame/oracle/src/lib.rs

```diff
@@ -69,11 +70,17 @@ pub mod pallet {
 	use sp_runtime::{
 		helpers_128bit::multiply_by_rational,
 		offchain::{http, Duration},
-		traits::{AtLeast32BitUnsigned, CheckedAdd, CheckedMul, CheckedSub, Saturating, Zero},
+		traits::{
+			AccountIdConversion, AtLeast32Bit, AtLeast32BitUnsigned, CheckedAdd, CheckedDiv,
+			CheckedMul, CheckedSub, Saturating, Zero,
+		},
 		AccountId32, ArithmeticError, FixedPointNumber, FixedU128, KeyTypeId as CryptoKeyTypeId,
-		PerThing, Percent, RuntimeDebug,
+		PerThing, Percent, RuntimeDebug, SaturatedConversion,
+	};
+	use sp_std::{
+		borrow::ToOwned, collections::btree_set::BTreeSet, fmt::Debug, str, vec, vec::Vec,
 	};
-	use sp_std::{borrow::ToOwned, fmt::Debug, str, vec, vec::Vec};
+	use std::ops::Add;
```

Comment:
remove

Reply:
Can you use the Add suggestion feature? Makes it easier to understand the context.

Reply:
ah note to my self, not to use `std` :D